docs/tracking.md

```diff
@@ -0,0 +1,192 @@
+
+As well as this entry of `remotes/$KERMIT`, Miss Piggy's monorepo will also gain an entry in its
+`config` file:
+
+```
+[remote "$THE_MUPPET_SHOW/$KERMIT"]
+        url = rad-p2p://$MISS_PIGGY@$KERMIT/$THE_MUPPET_SHOW.git?
+        fetch = +refs/heads/*:refs/remotes/$KERMIT/$THE_MUPPET_SHOW/*
+```
+
+The fetch refspec[5] here will seem off to you when comparing it to the above layout of a single
+monorepo, and you would be right. In this case Link only cares about the `url` and manages the fetch
+refspecs under the hood. The URL allows use to manage who we are tracking and for what URN.
+
+What we are learning from this exposition is that we add a peer as someone we track when we clone[3]
+their project. Miss Piggy is adding Kermit as a `remote`, and so she is tracking his changes. If
+Kermit announces changes and Miss Piggy's peer hears about these changes -- whether through Kermit's
+device or some other peer -- she will apply them to update her view of Kermit's remote.
+
+### Directly Track
+
+The other way a peer can track is by explicitly knowing a peer's identifier and telling their
+monorepo that they are interested in this peer. For example, Miss Piggy also heard that Rizzo the
+Rat is also taking part in the project and knows his peer identifier, $RIZZO. When she uses the
+`track`[4] function it creates a new entry in the monorepo config:
+
+```
+[remote "$THE_MUPPET_SHOW/$RIZZO"]
+        url = rad-p2p://$MISS_PIGGY@$RIZZO/$THE_MUPPET_SHOW.git?
+        fetch = +refs/heads/*:refs/remotes/$RIZZO/$THE_MUPPET_SHOW/*
+```
+
+The next time she hears from the network about changes from Rizzo she will fetch those changes and
+apply them to her monorepo.
+
+## The Tracking Exchange
+
+In the above example some extra details were elided because we were only talking about two peers. So
+when Miss Piggy cloned the repository from Kermit, it was almost obvious that he would end up in the
+`remotes`. But what if we talked about more peers?
+
+Let's say Gonzo ends up cloning from Miss Piggy. We can intuit from the above that Miss Piggy will
+end up as a remote in Gonzo's monorepo view of the project. But not only that, Kermit will also be
+added. Kermit's role is special since he's the maintainer of the project. In fact, when any peer
+clones a project from the network all the maintainers will end up in that peer's remote list, and so
+the maintainers are tracked. This makes sense since maintainers have the canonical view of the
+project. They're ensuring the health and consistency of this piece of work.
+
+The fun doesn't stop there. A portion of Miss Piggy's tracking graph will also be added to Gonzo's
+graph. The idea here is that Gonzo isn't just relying on who he cloned from and the maintainers, but
+also some portion of the network that is interested in the project. The natural question is, what
+portion of the network that Miss Piggy is tracking is also tracked by Gonzo? The answer is 2 degress
+out.
```

Comment:
Is this how it currently works? Or desired state in the future?

Reply:
Current workings